packages/suite/src/views/settings/general/Analytics.tsx

```diff
@@ -32,10 +34,8 @@ export const Analytics = () => {
                         dataTest="@analytics/toggle-switch"
                         isChecked={!!enabled}
                         onChange={() => {
-                            if (enabled) {
-                                return dispose();
-                            }
-                            enable();
+                            if (enabled) analytics.disable();
+                            else analytics.enable();
```

Comment:
nit: I found it much harder to read without the brackets, especially with the `else`. Let's do a short discussion on next code improvements meeting. So far leave it as you wish of course...

Reply:
655eacadac3da933adc7c4877ccf50c9aaf6d278

Reply:
https://www.notion.so/satoshilabs/blocks-without-brackets-or-not-5911948930024dc2a0f9d410dd488e2c